docker/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:14.04
+#QIIME basic installation env v0.1 
```

Comment:
Could you re-word this to say *QIIME base install* (as that's the specific term that we use to define this configuration)?